chia/policy/fee_estimator_demo.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict, Any, List
+
+from chia.policy.fee_estimation import FeeBlockInfo, FeeMempoolInfo
+from chia.types.mempool_item import MempoolItem
+
+from chia.full_node.fee_estimate import FeeEstimate
+from chia.util.ints import uint64
+
+MIN_MOJO_PER_COST = 5
+
+
+def demo_fee_rate_function(cost: int, time_in_seconds: int) -> uint64:
+    return uint64(cost * MIN_MOJO_PER_COST * max((3600 - time_in_seconds), 1))
+
+
+class FeeEstimatorDemo:  # FeeEstimatorInterface Protocol
```

Comment:
same question about not implementing the interface

Reply:
Hmm, I should remove this, or put it in a place where we can teach users about the codebase. Something like an `examples` directory.